src/main/webapp/js/auscope/Main-UI.js

```diff
@@ -331,6 +331,48 @@ Ext.application({
             ]
         });
         
+        var clearMapHandler = function() {
+
+            ActiveLayerManager.removeAllLayers(me.map);
+
+            me.map.setZoom(4);
+            var center = new portal.map.Point({longitude:133.3, latitude: -26});
+            me.map.setCenter(center);
+
+            /* set the Base Layer for the map */
+            Ext.each(me.map.layerSwitcher.baseLayers, function(baseLayer) {
+                if (baseLayer.layer.name === "Google Satellite") {
+                    me.map.map.setBaseLayer(baseLayer.layer);
+                    return false;
+                }
+            });
+
+            // if the browser supports local storage, clear the stored map state
+            if(typeof(Storage) !== "undefined") {
+                localStorage.removeItem("portalStorageApplicationState");
+                localStorage.removeItem("portalStorageDefaultBaseLayer");
+            }
+            portal.util.GoogleAnalytic.trackevent('ClearMapHandlerClick', 'Clear Map', 'http://portal.geoscience.gov.au');
+        };
+
+        var scannedMapsHandler = function() {
+
+            clearMapHandler();
+            var id = "250K-scanned-geological-maps";
+
+            var knownLayer = knownLayerStore.getById(id);
+            var scannedMapsLayer = me.layerFactory.generateLayerFromKnownLayer(knownLayer);
+            knownLayer.set('layer', scannedMapsLayer);
+
+            var filterParams = {bbox: {"westBoundLongitude":"102","southBoundLatitude":"…157","northBoundLatitude":"-2","crs":"EPSG:4326"}, opacity: 1}
```

Comment:
just checking but what is the southBoundLatitude "...157"?  Is it supposed to have dots before the number and are the numbers supposed to be hard coded?  All good if they are.